content/articles/postgresql-vs-mysql/index.md

```diff
@@ -0,0 +1,83 @@
+
+PostgreSQL also has disaster recovery features that include Point in Time Recovery ([PITR](https://dev.mysql.com/doc/mysql-backup-excerpt/8.0/en/point-in-time-recovery.html)), [tablespaces](https://www.postgresql.org/docs/10/manage-ag-tablespaces.html), and Write-Ahead Logging ([WAL](https://www.postgresql.org/docs/9.1/wal-intro.html)). It also supports [Synchronous](https://www.techopedia.com/definition/13739/synchronous-replication), Asynchronous and [Logical](https://www.postgresql.org/docs/10/logical-replication.html#:~:text=Logical%20Replication,-Table%20of%20Contents&text=Logical%20replication%20is%20a%20method,byte%2Dby%2Dbyte%20replication.) replications.
+
+PostgreSQL is also fully ACID compliant (Atomicity Consistency Isolation Durability).
+
+### Features of MySQL
+
+MySQL is a fast database for read-heavy data writes. It is an incredible choice for managing simple but bulk data projects. It is written in C and C++ and supports the languages C, C++ Perl, Java, [PHP](https://www.php.net/), Python, and [Tcl](https://www.tcl.tk/). MySQL applications can be developed on major operating systems, including [Windows](https://www.microsoft.com/en-us/windows), Symbian, [HP-UX](https://www.operating-system.org/betriebssystem/_english/bs-hpux.htm), [FreeBSD](https://www.freebsd.org/), [NetBSD](https://www.netbsd.org/), Linux, and [macOS](https://www.apple.com/macos/big-sur/).
+
+MySQL supports Open Database Connectivity ([ODBC](https://docs.microsoft.com/en-us/sql/odbc/reference/odbc-overview?view=sql-server-ver15)) a widely accepted application programming interface (API) for database access. With ODBC, an application can access different database management systems with the same source code.
+
+Just like PostgreSQL, MySQL supports replication in case of a hardware change. This also makes database queries faster.
+
+The database supports [InnoDB](https://dev.mysql.com/doc/refman/5.6/en/innodb-introduction.html) formats, which facilitates transactions in the system. The InnoDB formats also allow the creation of InnoDB tables. MySQL supports foreign key constraints for these tables hence eliminating unwanted cross-references in linked tables.
+
+The database is also equipped with JSON data validation, which rejects invalid JSON data from being entered into your projects.
+
+MySQL facilitates an easier transition to a new password in codes by supporting dual passwords for the entered data. A user does not need to edit an entire application when updating the password.
+
+Other features of MySQL include support for multi-version concurrency control, multi-threads using Kernel Threads, and huge data capability. The ANSI SQL standard database can handle rows exceeding 50 million.
+
+![ProstreSQL Vs. MySQL](/engineering-education/postgresql-vs-mysql.png)
+
```

Comment:
A table summarizing the two databases on the basis of features discussed in the article would help the reader gain more insights at the end of the article.